### DISCUSSION TITLE
Can multiple machines share a single monitor?

### DISCUSSION THREAD

**Discussion by CaptainKraft** [debauchee/barrier#1321] [Q&A]
There are two situations I'd like to make possible.

1. I have one monitor and two machines. Can I set up Barrier so that my single monitor can switch inputs between whichever machine I'd like to use.
2. I have two monitors and two machines, but I don't want one machine per monitor. I'd like both monitors to work for machine A while I'm using it, then I'd like to swap to machine B which will use both monitors.

Are either of these possible?

**Comment by ianjsikes** [comment]
Yes you can do this. Barrier will not handle actually switching the input of the monitors. You would either have to physically press a button on the monitors, or use something like [ddctl](https://github.com/kfix/ddcctl) to control them programmatically. You could set that up with AutoHotKey on Windows, and maybe BetterTouchTool, Karabiner, or skhd on Mac.
